tests/common/ixia/ixia_fixtures.py

```diff
@@ -193,11 +194,31 @@ def ixia_api(ixia_api_serv_ip,
     if api_session:
         api_session.assistant.Session.remove()
 
+
+@pytest.fixture(scope='module')
+def snappi_api(ixia_api_serv_ip,
+               ixia_api_serv_port):
+    """
+    Snappi session fixture for snappi Tgen API
+    Args:
+        ixia_api_serv_ip (pytest fixture): ixia_api_serv_ip fixture
+        ixia_api_serv_port (pytest fixture): ixia_api_serv_port fixture.
+    """
+    host = "https://" + ixia_api_serv_ip + ":" + str(ixia_api_serv_port)
+    api = snappi.api(host=host, ext="ixnetwork")
+
+    yield api
+
+    if api.assistant is not None:
```

Comment:
This is specific to IxNetwork, so add a check here. i.e. execute this statement only if assistant is a valid attribute of api.

Reply:
updated